tools/linter_lib/custom_check.py

```diff
@@ -576,17 +576,6 @@
         "good_lines": ["{{t 'translatable string' }}"],
         "bad_lines": ["{{t 'translatable string '}}"],
     },
-    {
-        "pattern": "placeholder='[^{]",
-        "description": "`placeholder` value should be translatable.",
-        "good_lines": [
-            '<input class="stream-list-filter" type="text" placeholder="{{ _(\'Filter streams\') }}" />'
-        ],
-        "bad_lines": ["<input placeholder='foo'>"],
-        "exclude": {
-            "templates/corporate",
-        },
```

Comment:
Why are these duplicates? I think they've got different ways of doing the quotes or something.

Reply:
Oh yeah! dropped this commit.